test/subscriptions.test.js

```diff
@@ -0,0 +1,51 @@
+import { app, h } from '../src';
+
+test('runs a subscription', () => {
```

Comment:
I don't know how much ceremony this test requires. Further more, this needs to be more thoroughly tested, including a test that explicitly tests cancel behaviour, and another for restart behaviour.

Reply:
@mrozbarry Yeah, that's why I'd prefer adding tests later.